astropy/modeling/tests/test_quantities_evaluation.py

```diff
@@ -42,7 +42,7 @@ def test_evaluate_with_quantities():
     # error is raised
     with pytest.raises(UnitsError) as exc:
         gq(1)
-    assert exc.value.args[0] == ("Gaussian1D: Units of input 'x', (dimensionless), could not be "
+    assert exc.value.args[0] == ("Units of input 'x', (dimensionless), could not be "
```

Comment:
This is realted to the `UnitsError` change in core.py.  If the `UnitsError` is restored then this test should pass as is.

Reply:
True for the rest of the differences in this module.